manifests/0000_90_console-operator_03_console_plugin_storage_migration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: migration.k8s.io/v1alpha1
+kind: StorageVersionMigration
+metadata:
+  name: console-plugin-storage-version-migration
+  annotations: 
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  resource:
+    group:  console.openshift.io
+    version: v1
+    resource: consoleplugins
+---
```

Comment:
Nit, probably don't need `---` at the end, and should have a newline char as the final character